### PR TITLE
Ch footer bug fixes

### PR DESCRIPTION
**Pull request by kip-guile**
## Description
Footer bug fix
## Type of change
Please delete options that are not relevant.
- [X] Bug fix (non-breaking change which fixes an issue)
## Checklist
Remove any items which are not applicable.
- [X] I have performed a self-review of my own code
- [X] My code follows the style guidelines of this project
- [X] My changes generate no new warnings

### CODE REVIEW

File: src/components/Auth/PublicRoute.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const PublicRoute = ({ component: Component, ...rest }) => (
       !localStorage.getItem('token') ? (
         <Component {...props} />
       ) : (
-        <Redirect to='/dashboard' />
+        <Redirect to='/' />
```
Comment (DimejiAre) [inline src/components/Auth/PublicRoute.js]: @kip-guile This component is preventing people that have logged in from going to register/login. It should redirect to their dashboard

**Review by vunderkind — APPROVED** [review]
Looks good to me. There are bugs present, but unrelated to the footer itself. Good stuff, Alex!

File: src/components/DashboardHome/Dashboard.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ const Dashboard = () => {
         <Content />
       </div>
 
-      <Footer />
+      {/* <Footer /> */}
```
Comment (aapetsi) [inline src/components/DashboardHome/Dashboard.js:31]: we dont need commented out code. please delete them

File: src/components/DashboardHome/nav.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+function Nav () {
+  return (
+    <div className='nav' style={{ backgroundColor: 'white' }}>
+      <ul id='nav_ul'>
+        <li className='li'><NavLink to='dashboard' activeClassName='activeLink'><Icon className='icon' type='home' /><p>Home</p></NavLink></li>
+        <li className='li'><NavLink to='inventory' activeClassName='activeLink'><Icon className='icon' type='shop' /><p>Store</p></NavLink></li>
+        <li id='add'><NavLink className='navlink' to='createItem'><Icon className='icon' theme='filled' type='plus-circle' /></NavLink></li>
+        <li className='li'><Icon className='icon' type='wallet' /><p>Account</p></li>
+        <li className='li'><NavLink to='profile' activeClassName='activeLink'><Icon className='icon' type='user' /><p>Profile</p></NavLink></li>
+      </ul>
```
Comment (aapetsi) [inline src/components/DashboardHome/nav.js:15]: check the definition of the to property on the NavLink

**Review by aapetsi — APPROVED** [review]

**Pull request merged**